doc/requirements/sequential_cutoff

```diff
@@ -0,0 +1,234 @@
+---
+group: Sequential cutoff
+---
+
+Sequential cutoff feature allows to detect sequential data streams
+and redirect them to backend volume (bypassing the cache).
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting sequential cutoff policy for core
+id: policy_core
+---
+
+The management tool shall allow to specify sequential cutoff policy
+for given core.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting sequential cutoff policy for all cores in given cache instance
+id: policy_all_cores
+---
+
+The management tool shall allow to specify sequential cutoff policy
+for all cores associated with given cache instance at once.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed sequential cutoff policies
+id: allowed_policies
+---
+
+There shall be three sequential cutoff policies:
+
+1. ALWAYS
+2. FULL
+3. NEVER
```

Comment:
Referring to the previous source, please use lower case names for these policies everywhere.

Reply:
Fixed.